### PR TITLE
Moved queries to respective controllers

### LINKED ISSUE #2: Refactor API endpoints
- Right now, all relevant end points are present on the 'Runs' controller
- To add more cohesion, endpoints relevant to family and accession data should be migrated from the 'Runs' controller to their respective controllers
- Current Endpoint for Accessions: http://serratustest-dev-sumdb.us-east-1.elasticbeanstalk.com/api/runs/get-runs/NC_032111.1
- Example: http://serratustest-dev-sumdb.us-east-1.elasticbeanstalk.com/api/accession/get-accessions/NC_032111.1

### PR DESCRIPTION
**Pull request by sepetaana**
- Moved queries to respective controllers
- Fixed spacing 
- Added SerratusApi sln

### CODE REVIEW

**Comment by victorlin** [comment]
why the 2nd solution file? shouldn't we use the root-level solution file to see both projects at once?

File: SerratusApi/Controllers/RunsController.cs [hunk 1]
```diff
@@ -28,55 +28,34 @@ public async Task<ActionResult<IEnumerable<Run>>> GetRuns()
...
         }
         
-        [HttpGet("get-run/{sra}")]
-        public async Task<ActionResult<Run>> GetSummary(string sra)
+        [HttpGet("get-runs/{run}")]
+        public async Task<ActionResult<Run>> GetSummaryForSraAccession(string run)
         {
 
-            var run = await _context.Runs.FirstOrDefaultAsync(r => r.Sra == sra);
+            var sraAccessions = await _context.Runs.FirstOrDefaultAsync(r => r.Sra == run);
```
Comment (victorlin) [inline SerratusApi/Controllers/RunsController.cs]: `sraAccession` would be a better name (singular)

**Review by victorlin — CHANGES_REQUESTED** [review]

File: SerratusApi/Controllers/RunsController.cs (hunk 1 above)
Comment (lohrd) [reply SerratusApi/Controllers/RunsController.cs]: Agreed

**Review by lohrd — APPROVED** [review]

**Review by victorlin — APPROVED** [review]

**Comment by victorlin** [comment]
ref #2

**Pull request merged**